topics/metabolomics/tutorials/gcms/tutorial.md

```diff
@@ -0,0 +1,432 @@
+> > {: .text-justify}
+> > 
+> {: .solution}
+> <br>
+> **3** - What is the size (in MB) of your final RData file ?
+> > ### {% icon solution %} Solution
+> > 
+> > To be able to see the size of a file in your history, you just have to select it. It will deploy informations about it and you can see the size of yours. For our example, the size of the final file is **1.4 MB**.
+> > {: .text-justify}
+> > 
+> {: .solution}
+{: .question}
+
+
+# Processing with metaMS part
+
+**metaMS** is a R package for MS-based metabolomics data. It can does basic peak picking and grouping using functions from **XCMS** and **CAMERA** packages. The main output of **metaMS** is a table of feature intensities in all samples which can be analysed with multivariate methods immediately. The package also offers the possibility to create in-house databases of mass spectra (including retention information) of pure chemical compounds. These databases can then be used for annotation purposes. The most important functions of this package are *runGC* and *runLC* (and each one to create databases *createSTDdbGC* and *createSTDdbLC*).
+{: .text-justify}
+During this tutorial we are interested in GC-MS analysis, so we will use the *runGC* function of **metaMS** and described it in details to be able to understand this function. The standard workflow of **metaMS** for GC-MS data is the following : 
+
+![Workflow metaMS](../../images/tuto_gcms_workflow_metaMS.png "Workflow of metaMS for GC datas")
+
+The *runGC* function is implemented in **metaMS.runGC {% icon tool %} tool** in W4M Galaxy. It takes a vector of file names, corresponding to the samples, and a settings list as mandatory arguments. In addition, some extra arguments can be provided. In particular, a database of standards, as discussed later in tutorial, can be provided for annotation purposes. This tool regroups all these steps that are described in the following parts to be able to understand all its functionnalities and particularities. We will run the tool after we understand each of its steps because it is important to know what are the best parameters for our datas and why each parameter is done. 
+{: .text-justify}
+
+
+## Peak picking
+
+The peak picking is performed by the usual **XCMS** functions. A function has been written in **metaMS** to allow the individual parameters to be passed to the function as a settings list. The result is that the whole of the **XCMS** functionality is available, simply by changing the values of some settings, or by adding fields. 
+ {: .text-justify}
+Whereas the package is not up-to-date since the new version of **XCMS** (3.x). This new version brought a lot of new object and transformed the peak picking processus. To have the last version of this processus, **metaMS** authorized **to start its function directly with the file containing all peak picking results**. 
```

Comment:
```suggestion
Whereas the package is not up-to-date since the new version of **XCMS** (3.x). This new version brought a lot of new objects and transformed the peak picking process. To have the last version of this process, **metaMS** authorized **to start its function directly with the file containing all peak picking results**. 
```